roofit/roofitcore/src/RooFitLegacy/RooHashTable.cxx

```diff
@@ -32,7 +32,7 @@ ClassImp(RooHashTable);
 /**
 \file RooHashTable.cxx
 \class RooHashTable
-\ingroup Roofitcore
+\ingroup Roofitlegacy
```

Comment:
Did you define this group? Doxygen needs an explicit
```
/// \group Roofitlegacy
/// Description 1st sentence.
/// Description ...
```
to make something reasonable of it.

Reply:
It seems the group is already defined: https://root.cern/doc/master/group__Roofitlegacy.html